xtask/src/tools/git.rs

```diff
@@ -132,6 +132,22 @@ impl GitRunner {
         Ok(())
     }
 
+    // takes a PackageTag and creates an empty commit after publish
+    pub(crate) fn empty_commit(&self, package_tag: &PackageTag, dry_run: bool) -> Result<()> {
```

Comment:
Does this push to main? If so do we have permissions to push directly to main?

Reply:
ah that's a good point. i looked in the settings and I think it's just me and @abernix that can. perhaps a different strategy is better here.